Lib/configparser.py

```diff
@@ -895,13 +913,20 @@ def write(self, fp, space_around_delimiters=True):
         if self._defaults:
             self._write_section(fp, self.default_section,
                                     self._defaults.items(), d)
+        if UNNAMED_SECTION in self._sections:
```

Comment:
Sorry for not noticing this earlier, but I think there still is a bug here: If a config contains both a DEFAULT and an unnamed section, the headerless unnamed section must be written *before* the DEFAULT section, while currently it's the other way around.

Reply:
Sorry for this mistake!